apps/crn-server/src/controllers/research-outputs.ts

```diff
@@ -80,6 +80,30 @@ export default class ResearchOutputs implements ResearchOutputController {
     };
   }
 
+  async fetchDrafts(
```

Comment:
looks like a copy of the above method, do you really need a whole new method for this?

Reply:
removed function